front/tests/005-operational-studies.spec.ts

```diff
@@ -8,9 +9,21 @@ import PlaywrightScenarioPage from './pages/scenario-page-model';
 import { ProjectPage } from './pages/project-page-model';
 import { StudyPage } from './pages/study-page-model';
 
-test.describe('Testing if all mandatory elements simulation configuration are loaded in operationnal studies app', () => {
-  test('Testing pathfinding with rollingstock an composition code', async ({ page }) => {
-    test.setTimeout(90000); // 1min30
+const createdInfra = process.env.E2ETESTS_SMALL_INFRA;
+const createdProject = process.env.E2ETESTS_PROJECT;
+const createdStudy = process.env.E2ETESTS_STUDY;
+const createdScenario = process.env.E2ETESTS_SCENARIO;
+const createdRollingStock1 = process.env.E2ETESTS_ROLLING_STOCK_1;
+
+// TODO: remove (enabled) when every tests are refactored
+test.describe('Testing if all mandatory elements simulation configuration are loaded in operationnal studies app (enabled)', () => {
+  test('Testing pathfinding with rollingstock and composition code', async ({ page }) => {
+    const testInfra: Infra = JSON.parse(createdInfra!);
+    const newProjectData: Project = JSON.parse(createdProject!);
+    const newStudyData: Study = JSON.parse(createdStudy!);
+    const newScenarioData: Scenario = JSON.parse(createdScenario!);
+    const rollingStock1: RollingStock = JSON.parse(createdRollingStock1!);
```

Comment:
Parse the data outside the test and rename the variables
- infra, project, study, scenario